fuzzbench/fake_jobs.py

```diff
@@ -33,3 +33,8 @@ def run_trial():
 def measure_corpus_snapshot():
     """Measure a corpus snapshot."""
     return True
+
+
+def fake_job():
+    """A signal fake job function."""
+    return True
```

Comment:
why do we need another fake job function? why don't we build images?

Reply:
> why do we need another fake job function?

Since this function is not related to experiment logic, I think another different name will be helpful for readability.

> why don't we build images?

As said, build image fake job will be confusing here.